css/bootstrap-slider.css

```diff
@@ -243,4 +244,75 @@
   background-repeat: repeat-x;
   filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff89cdef', endColorstr='#ff81bfde', GradientType=0);
   opacity: 1;
-}
+  background-color:#89cdef;
```

Comment:
you need to add these to the LESS and SASS files (which is the source that compiles into the css)